Libraries/Image/Image.ios.js

```diff
@@ -105,6 +134,24 @@ var Image = React.createClass({
     validAttributes: ReactIOSViewAttributes.UIView
   },
 
+  onLoadingStart: function(event: Event) {
+    if (this.props.onLoadingStart) {
+      this.props.onLoadingStart(merge(event.nativeEvent, { target: this }));
```

Comment:
Good point! Also I just remembered that mutating React events isn't a good idea since they're re-used from a pool and the values could be bogus by the time they're accessed.